plugin-api/src/main/java/org/hyperledger/besu/plugin/data/BlockBody.java

```diff
@@ -46,4 +46,11 @@ public interface BlockBody {
    * @return The list of withdrawals of the block.
    */
   Optional<? extends List<? extends Withdrawal>> getWithdrawals();
+
+  /**
+   * Returns the list of deposits of the block.
+   *
+   * @return The list of deposits of the block.
+   */
+  Optional<? extends List<? extends Deposit>> getDeposits();
```

Comment:
Until this EIP is shipped on mainnet we should add the `@Unstable` annotation, implying it may change or go away without warning.  For all new fields in the plugin-api package.